dialect/postgresql.go

```diff
@@ -25,7 +25,7 @@ func (d postgreSQL) EncodeBool(b bool) string {
 }
 
 func (d postgreSQL) EncodeTime(t time.Time) string {
-	return MySQL.EncodeTime(t)
+	return `'` + t.Format(time.RFC3339Nano) + `'`
```

Comment:
Could you update dialect.go timeFmt instead so that it is available in all dialects?

Reply:
I tested this format with the rest of the bases, MySQL (5.6) and MS SQL (2017) cannot parse time in this format. I suggest using the new format for PostgreSQL, leaving the old implementation for MySQL, SQLite and MS SQL.